lib/src/core/signal_client.dart

```diff
@@ -223,6 +236,9 @@ class SignalClient extends Disposable with EventsEmittable<SignalEvent> {
       case lk_rtc.SignalResponse_Message.notSet:
         logger.info('signal message not set');
         break;
+      case lk_rtc.SignalResponse_Message.pong:
+        _resetPingTimeout();
```

Comment:
should it be `_clearPingTimeout` here? that means ping had been successful. without setting a new one.

Reply:
oops sorry.. I think I'm confusing myself.

Reply:
I think either doing _clearPingTimeout or _resetPingTimeout will work

If the pingTimer is cleared, a new one will be created when sendPing
If we keep _resetPingTimeout, we don't need to call _resetPingTimeout again when we sendPing